.dockerignore

```diff
@@ -305,11 +305,20 @@ dmypy.json
 
 **/node_modules
 .idea
+.coverage*
 
 
 ######### NOTE: stuff above is from .gitignore
 ######### Please keep in sync!
 .git
 .ci  # otherwise build context is invalidated over and over again
 
+.circleci
+.github
+benchmarks
+ci
+doc
+docker/user_data
+docker/docker_files/indexer-config.py.example
+extension
```

Comment:
I think best to keep these -- they are used (or can potentially be used by the CI). It fails now due to extension/.ci/build missing https://github.com/karlicoss/promnesia/actions/runs/7427827283/job/20327375203
https://github.com/karlicoss/promnesia/blob/master/.ci/end2end_tests.Dockerfile#L64

Reply:
I didn't think about the impact this would have on CI - I've now pushed changes to use a less lazy way of reducing the size of the docker image